### ISSUE TITLE
Proguard warnings in debug

### ISSUE THREAD

**Issue by AlbertVilaCalvo** [21Buttons/FragmentTestRule#10]
When I enable proguard in debug builds, I get this warnings:

```
Warning:com.android21buttons.fragmenttestrule.FragmentTestRule: can't find referenced method 'void runOnUiThread(java.lang.Runnable)' in program class com.android21buttons.fragmenttestrule.FragmentTestRule
Warning:com.android21buttons.fragmenttestrule.FragmentTestRule: can't find referenced method 'android.app.Activity getActivity()' in program class com.android21buttons.fragmenttestrule.FragmentTestRule
Warning:com.android21buttons.fragmenttestrule.FragmentTestRule$1: can't find referenced method 'android.app.Activity getActivity()' in program class com.android21buttons.fragmenttestrule.FragmentTestRule
```

To compile I have to add `-dontwarn com.android21buttons.fragmenttestrule.*` to my proguard-rules.pro.

Would it be possible to have the proguard rules included in the library itself so that the each user doesn't have to add them manually? Maybe using `consumerProguardFiles`.

**Comment by BraisGabin** [comment]
I think that this is because we must add de library as `debugCompile`. A new good reason to split the library in two components...

**Comment by AlbertVilaCalvo** [comment]
If you are not going to split the library, please consider adding the proguard rules so that each user doesn't have to add them.

**Comment by BraisGabin** [comment]
The slit is done. I'll publish a new version of the library soon.

**Comment by BraisGabin** [comment]
Done. The new version will be up in a couple of hours. If the issue persists, please, reopen.
